cnd/src/http_api/action.rs

```diff
@@ -34,7 +34,8 @@ pub enum ActionExecutionParameters {
 pub enum ActionResponseBody {
     BitcoinSendAmountToAddress {
         to: bitcoin_support::Address,
-        amount: bitcoin_support::BitcoinQuantity,
+        #[serde(with = "bitcoin_support::amount::serde::as_sat")]
```

Comment:
This is strange, we have a serialization for struct `SendToAddress` and for this `ActionResponseBody`.